yamcs-client/src/yamcs/model.py

```diff
@@ -255,9 +255,23 @@ def out_count(self):
         """The number of outbound data events (example: command count)."""
         return self._proto.dataOutCount
 
+    @property
+    def actions(self):
+        """Custom actions."""
+        return self._proto.actions
+
+    @property
+    def extra(self):
+        """Custom info fields."""
+        return self._proto.extra
```

Comment:
Convert to a true Python dictionary instead of the protobuf generated code Cfr. extra field in `yamcs.model.Event`.